pkg/config/config.go

```diff
@@ -36,7 +36,7 @@ func New() Config {
 
 		// these are used when the image stream does not exist and created for the first time by the operator
 		MDCImageStreamInitialImage:        getEnv("MDC_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/aerogear/mobile-developer-console:1.1.11"),
-		OauthProxyImageStreamInitialImage: getEnv("OAUTH_PROXY_IMAGE_STREAM_INITIAL_IMAGE", "docker.io/openshift/oauth-proxy:v1.1.0"),
+		OauthProxyImageStreamInitialImage: getEnv("OAUTH_PROXY_IMAGE_STREAM_INITIAL_IMAGE", "quay.io/openshift/origin-oauth-proxy:4.2.0"),
```

Comment:
Hey. I'm not sure about doing this change here.  It won't get pulled in by the version 0.2.6 I think we'd need to bump the version and generate a new CSV?

Reply:
True, I agree that we need to bump the version.

Reply:
Let's not worry about bumping the version again before having this merged -- ideally this will be merged _before_ the next release, so that we can do the new CSV as part of the new release.